catalog/video/catalog-mpegts-transformer/src/main/resources/videoMetacardType.json

```diff
@@ -0,0 +1,284 @@
+{
+  "metacardTypes": [
+    {
+      "type": "VideoMetacardType",
```

Comment:
We need to discuss the contents of the VideoMetacardType.  I'm hoping we can use more of the existing "basic" attributes and add in only what we need for video.